docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3.8"
+
+services:
+  acezero:
+    restart: always
+    container_name: acezero
+    build:
+      context: .
+      dockerfile: Dockerfile
+    tty: true
+    stdin_open: true
+    ports:
+      - "7771:7007"
+    volumes:
+      - ./images:/workspace/images
```

Comment:
Suggestion: remove this, it's not needed for the final container (unless I'm mistaken)